### DISCUSSION TITLE
Ugs

### DISCUSSION THREAD

**Discussion by yitayalyonas** [winder/Universal-G-Code-Sender#2217] [Help]
Hi guys!I am using ugsplatform(version  2.0.15)  when i connect ugs , it connects and the sign becomes orange color  and sometimes working fine but sometimes also jog controller not active 
Please help me?

**Comment by breiler** [comment]
What operating system are you using? What controller firmware and version are you using?

If the controller is in an alarm state the jog controller will be inactive, is the jog controller inactive even when in idle state?
